keps/sig-node/2254-cgroup-v2/README.md

```diff
@@ -52,6 +63,186 @@ This proposal aims to:
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+N/A.  Not relevant to upgrades.  If the host is running with cgroup v2 then
+it will be automatically detected and used.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [ ] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
+  - Components depending on the feature gate:
+- [X] Other
+  - Describe the mechanism:
+    configure the hosts to use cgroup v2
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+    No, each host can be restarted to cgroup v2 separately
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+    It requires downtime of a node since it needs to be rebooted
+
+###### Does enabling the feature change any default behavior?
+
+N/A.  It must work in the same way as on cgroup v1
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+yes, it is enough to restart the node on cgroup v1
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+It should work seamlessly without any difference
+
+###### Are there any tests for feature enablement/disablement?
+
+The same E2E tests that work on cgroup v1 should work on cgroup v2
+
+### Rollout, Upgrade and Rollback Planning
+
+N/A.  Each node can be configured separately.
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+N/A.  It requires a reboot to be enabled.  If the workload accesses directly the
+cgroup file system, then also the workload must be enabled for cgroup v2.
+
+###### What specific metrics should inform a rollback?
+
+Pods not being healthy. One could inspect if the pods are getting the cgroups
+set correctly referencing the conversion table in this KEP.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+N/A.  It depends on the node configuration and it is stateless.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+The cgroup file system inside of the containers will use cgroup v2 instead of cgroup v1.
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+An operator could run `cat /proc/self/cgroup` on a node to check if it is running in cgroups v2 mode.
```

Comment:
Ok. Please consider in the future (GA) making this sort of information available via the API or a metric. Like in Node status or something. Because if I am monitoring 1000s of clusters with 1000s of nodes each, this is not convenient. I won't block this now though for that.

Reply:
Thanks!